Classes/Notifications/NotificationsViewController.swift

```diff
@@ -214,30 +245,7 @@ FlatCacheListener {
     // MARK: FlatCacheListener
 
     func flatCacheDidUpdate(cache: FlatCache, update: FlatCache.Update) {
-        switch update {
-        case .item(let item):
-            // only handle NotificationViewModels
-            if let notification = item as? NotificationViewModel {
-                var models = [ListDiffable]()
-                // do a pass over each model and search for the updated item
-                for model in self.models {
-                    // if not showing read items and the models match (id only)
-                    if let currentNotification = model as? NotificationViewModel,
-                        currentNotification.id == notification.id {
-                        // swap the model if not read, otherwise exclude it
-                        // this powers the "swipe to archive" feature deleting the cell
-                        if showRead || !notification.read {
-                            models.append(notification)
-                        }
-                    } else {
-                        models.append(model)
-                    }
-                }
-                self.update(models: models, animated: true)
-                self.resetLeftBarItem()
-            }
-        case .list: break
-        }
+        self.update(animated: true)
```

Comment:
This needs to only update on `.item` otherwise it'll update for _every_ object when refreshed.